<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SynthesisCorrectWidget</class>
 <widget class="QWidget" name="SynthesisCorrectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page_1">
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>里程_帧号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
